src/main/resources/database/schema.sql

```diff
@@ -23,31 +24,39 @@ create table if not exists dataset (
    Primary Key (dataset_id)
 );
 
```

Comment:
![image](https://user-images.githubusercontent.com/19560266/153223012-693d714b-244c-4f43-9da6-8f7a65f33997.png)

Reply:
Include in another ticket the creation of the relationship between ```dataset``` and ```record``` table.